.github/workflows/skywalking.yaml

```diff
@@ -18,17 +18,18 @@ name: CI
 
 on:
   pull_request:
-    paths-ignore:
-      - "*.md"
-      - "*.txt"
-      - "skywalking-ui"
-      - ".asf.yaml"
-      - ".dlc.yaml"
-      - ".licenserc.yaml"
-      - "docs/menu.yml"
-      - ".github/workflows/codeql.yaml"
-      - "dist-material/release-docs"
-      - "*/component-libraries.yml"
+    paths:
```

Comment:
What does this `paths-ignore` to `path` mean? The following patterns should be CI irrelative?

Reply:
There are some issues in the conditions to run / not to run the workflows, will take another look, just reverted these